src/syseng/magfrm.19

```diff
@@ -86,8 +86,11 @@ WHMCHN:	TYP FOR WHICH MACHINE?
 		  .VALUE
 		 JRST .+1 ]
 	MOVSI A,-LMCHNTB	
+ifn 0,[	;; BV: The original program checks to see if the user typed a known machine,
+    	;; but I argue there is no point in doing that.
```

Comment:
I think I may agree with you on that.  And if so, MCHNTB isn't needed either.

There is some value in checking that there is a corresponding MADGMP BIN file for the target ITS but the .OPEN will fail if there isn't.